packages/containers/psammead-timestamp-container/src/timestampUtilities.js

```diff
@@ -21,6 +21,13 @@ moment.relativeTimeThreshold('M', 12);
 
 const defaultFormat = 'LL, LT z';
 
+export const formatDuration = (durationValue, format = 'mm:ss') => {
```

Comment:
We need to bump the version of `psammead-timestamp-container` since we've added some functionality